src/HatelineModule.cs

```diff
@@ -1,110 +1,146 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
+using System.Xml;
 using Celeste.Mod.Hateline.CelesteNet;
 using FMOD.Studio;
 using Monocle;
 using MonoMod.ModInterop;
 
-namespace Celeste.Mod.Hateline
-{
-    public class HatelineModule : EverestModule
-    {
-        public static HatelineModule Instance { get; private set; }
-
-        public override Type SettingsType => typeof(HatelineModuleSettings);
-        public static HatelineModuleSettings Settings => (HatelineModuleSettings)Instance._Settings;
+namespace Celeste.Mod.Hateline;
 
-        public override Type SessionType => typeof(HatelineModuleSession);
-        public static HatelineModuleSession Session => (HatelineModuleSession)Instance._Session;
-
-        public bool HasForcedHat => Settings.AllowMapChanges && Session?.MapForcedHat != null;
+public class HatelineModule : EverestModule
+{
+    public static HatelineModule Instance { get; private set; }
 
-        public bool ShouldShowHat => HasForcedHat || Settings.Enabled;
-        public string? CurrentHat => HasForcedHat ? Session.MapForcedHat : Settings.SelectedHat;
+    public override Type SettingsType => typeof(HatelineModuleSettings);
 
-        public int CurrentX => HasForcedHat ? Session.mapsetX : Settings.CrownX;
-        public int CurrentY => HasForcedHat ? Session.mapsetY : Settings.CrownY;
+    public static HatelineModuleSettings Settings => (HatelineModuleSettings)Instance._Settings;
 
-        public static List<string> hats = new List<string>();
+    public override Type SessionType => typeof(HatelineModuleSession);
 
-        public const string DEFAULT = "none";
+    public static HatelineModuleSession Session => (HatelineModuleSession)Instance._Session;
 
-        public HatelineModule()
-        {
-            Instance = this;
-        }
+    public HatelineModule()
+    {
+        Instance = this; 
+#if DEBUG
+        // debug builds use verbose logging
+        Logger.SetLogLevel(nameof(HatelineModule), LogLevel.Verbose);
+#else
+        // release builds use info logging to reduce spam in log files
+        Logger.SetLogLevel(nameof(KelperModule), LogLevel.Info);
```

Comment:
👀 :3c

Reply:
I'm not sure what this comment means.

Reply:
`nameof(HatelineModule)` but then `nameof(KelperModule)` :v
What happened there. Wait how does that even compile.

Reply:
I see. Yea that's a funny typo. Can't tell if it was copypasting or just force of habit. Will fix.